.tito/custom.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# (using src/tito/builder/fetch.py from the tito project as a template)
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+#
+
```

Comment:
Please make this file follow our new python guidelines:

http://clusterlabs.org/doc/en-US/Pacemaker/1.1-pcs/html-single/Pacemaker_Development/index.html#idm47234057985456

Do we want these files in the distribution tarball?